Makefile

```diff
@@ -26,6 +26,10 @@ current_dir := $(patsubst %/,%,$(dir $(mkfile_path)))
 docker_env_file := "$(current_dir)/docker.env"
 unstables := $(cat .circleci/unstable_tests.txt | tr '\n' :)
 
+# tests need these environment variables to be unset
+OPENQA_BASEDIR =
+OPENQA_CONFIG =
```

Comment:
So that explains what I was seeing. I switched from `openqa-test` to `make test TESTs=` for everything recently and got confused why the real basedir showed up in test logs. :facepalm: